src/ensembl/src/content/home/Home.tsx

```diff
@@ -15,25 +15,160 @@
+                  <div className={styles.bottomText}>
+                    Look at genes & transcripts in their genomic context
+                  </div>
+                </div>
+              </div>
+
+              <div className={styles.selectionContainer}>
+                <div className={styles.buttonsContainer}>
+                  <Link to="/entity-viewer">
+                    <div className={styles.button}>
+                      <span>Entity viewer</span>
+                      <EntityViewerIcon className={styles.icon} />
+                    </div>
+                  </Link>
+                </div>
+                <div className={styles.bottomTextContainer}>
+                  <div className={styles.bottomText}>
+                    Get gene & transcript information
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          {isEnvironment([Environment.DEVELOPMENT, Environment.INTERNAL]) ? (
+            <div className={styles.homeTopRight}>
+              <Link to="/help">{helpButton}</Link>
+            </div>
+          ) : (
+            <div
+              className={classNames(
+                styles.homeTopRight,
+                styles.homeTopRightDisabled
+              )}
+            >
+              {helpButton}
+            </div>
+          )}
+        </div>
+
+        <div className={styles.speciesContainer}>
+          <span className={styles.platform}></span>
+          <img src={speciesStripUrl} className={styles.speciesStrip} />
+        </div>
+      </div>
+      <div className={styles.footer}>
+        <div className={styles.footerLeft}>
+          <Logotype className={styles.logotype} />
+          <img src={ebiLogoUrl} className={styles.ebiLogo} />
+          <div className={styles.text}>
+            Wellcome Genome Campus, Hinxton, Cambridgeshire CB10 1SD, UK
+          </div>
+        </div>
+
+        <div className={styles.footerRightIcons}>
+          <a
+            href="//www.ensembl.info"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <div className={styles.link}>Ensembl blog</div>
+          </a>
+          <a
+            href="//www.ensembl.info"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img src={blogIconUrl} className={styles.mediaIcons} />
+          </a>
+          <a
+            href="//www.facebook.com/Ensembl.org"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img src={facebookIconUrl} className={styles.mediaIcons} />
+          </a>
+          <a
+            href="//www.twitter.com/ensembl"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img src={twitterIconUrl} className={styles.mediaIcons} />
+          </a>
+          <a
+            href="//www.elixir-europe.org/platforms/data/core-data-resources"
```

Comment:
Elixir doesn't have a `www` in the url:

![image](https://user-images.githubusercontent.com/6834224/123655797-3df93b00-d827-11eb-8696-2c620bfaeef9.png)

Reply:
Done. same as above

Reply:
yep, Elixir also does a redirect:

![image](https://user-images.githubusercontent.com/6834224/123939462-0f4aa400-d990-11eb-80df-291c22860bee.png)